apps/server/src/modules/learnroom/uc/course-info.uc.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { AuthorizationService } from '@modules/authorization';
+import { ClassService } from '@modules/class';
+import { classFactory } from '@modules/class/domain/testing';
+import { ClassesRepo } from '@modules/class/repo';
+import { GroupService } from '@modules/group';
+import { GroupRepo } from '@modules/group/repo/';
+import { SchoolService } from '@modules/school';
+import { schoolFactory } from '@modules/school/testing';
+import { UserService } from '@modules/user';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Page } from '@shared/domain/domainobject';
+import { Permission, RoleName, SortOrder } from '@shared/domain/interface';
+import { groupFactory, setupEntities, UserAndAccountTestFactory, userDoFactory, userFactory } from '@shared/testing';
+import { Course, COURSE_REPO, CourseRepo as CourseDORepo, CourseSortQueryType, CourseStatusQueryType } from '../domain';
+import { CourseDoService } from '../service';
+import { courseFactory as courseDoFactory } from '../testing';
+import { CourseInfoUc } from './course-info.uc';
+import { CourseInfoDto } from './dto';
+
+describe('CourseInfoUc', () => {
+	let module: TestingModule;
+	let uc: CourseInfoUc;
+	let groupRepo: DeepMocked<GroupRepo>;
+	let classesRepo: DeepMocked<ClassesRepo>;
+
+	let authorizationService: DeepMocked<AuthorizationService>;
+	let schoolService: DeepMocked<SchoolService>;
+	let courseDoService: DeepMocked<CourseDoService>;
+	let groupService: DeepMocked<GroupService>;
+	let userService: DeepMocked<UserService>;
+	let classService: DeepMocked<ClassService>;
+
+	beforeAll(async () => {
+		await setupEntities();
+		module = await Test.createTestingModule({
+			providers: [
+				CourseInfoUc,
+				{
+					provide: AuthorizationService,
+					useValue: createMock<AuthorizationService>(),
+				},
+				{
+					provide: SchoolService,
+					useValue: createMock<SchoolService>(),
+				},
+
+				{
+					provide: CourseDoService,
+					useValue: createMock<CourseDoService>(),
+				},
+				{
+					provide: COURSE_REPO,
```

Comment:
you still provide repos here that you don't use. Check your provides with the injects in the uc.